### DISCUSSION TITLE
Structured data about and within documents

### DISCUSSION THREAD

**Discussion by almereyda** [outline/outline#3613] [Ideas]
A knowledge base's utility is strongly defined by accessibilty and discoverability of its content. While some people like to organise subjects hierarchically, as with nested collections #3591 of documents and sub-documents, others have requested a way for categorising documents through #765 #2256 or topologically through a visualisation of the internal link network #1692.

Additionally, coming from YAML-frontmatter- and Logseq-properties-land, we may have gotten used to enriching documents with forms of structured data other than tags, that can be queried, consumed and modified.

> As an editor of an Outline document, I need to able to add structured data to documents, in order to provide account and issue numbers, URLs of related resources and other metadata about the document.

**Comment by michael-rowe** [comment]
This was literally the first thing I looked for when I started playing around. I'm interested in publishing the documentation we produce, and in the academic community, there's an expectation that we can at least specify keywords / tags.
